example-app/src/app.ts

```diff
@@ -52,6 +52,11 @@ async function main() {
   app.use(stagingArea());
   app.use(sourceTx());
 
+  app.multiple(
```

Comment:
Shouldn't we either have this line or 60? What happens if there are overlapping routes?